src/lib/ikea.ts

```diff
@@ -825,3 +838,87 @@ export function addCustomClusterManuSpecificIkeaUnknown(): ModernExtend {
         commandsResponse: {},
     });
 }
+
+const unfreezeMechanisms: {
+    [key: string]: (entity: Zh.Endpoint | Zh.Group) => Promise<void>;
+} = {
+    // WS lights:
+    //   Aborts the color transition midway: light will stay at the intermediary
+    //   state it was when it received the command.
+    // Color lights:
+    //   Do not support this command.
+    moveColorTemp: async (entity) => {
+        await entity.command("lightingColorCtrl", "moveColorTemp", {rate: 1, movemode: 0, minimum: 0, maximum: 600}, {});
+    },
+
+    // WS lights:
+    //   Same as "moveColorTemp".
+    // Color lights:
+    //   Finishes the color transition instantly: light will instantly
+    //   "fast forward" to the final state, post-transition.
+    genLevelCtrl: async (entity) => {
+        await entity.command("genLevelCtrl", "stop", {}, {});
+    },
+};
+
+// zigbee commands which will freeze an IKEA light
+const willFreeze = (clusterKey: string | number, payload: KeyValue): payload is KeyValue & {transtime: number} =>
+    payload &&
+    // any color command with a transition will freeze the light...
+    (payload.transtime as number) > 0 &&
+    clusterKey === "lightingColorCtrl" &&
+    // ...except for 'stop' commands:
+    payload.rate !== 1 &&
+    payload.movemode !== 0;
+
+// keys for whom we need to unfreeze the light before issuing a corresponding command to said light
+const keysNeedingUnfreeze = new Set(["state", "brightness", "brightness_percent", "color_temp", "color_temp_percent"]);
+
+const ikea_bulb_unfreeze = (next: Tz.Converter["convertSet"]) => {
+    const converter: Tz.Converter["convertSet"] = async (entity, key, value, meta) => {
+        if (!keysNeedingUnfreeze.has(key)) {
+            return await next(entity, key, value, meta);
+        }
```

Comment:
@lbschenkel I'm not sure about this change I've introduced - I guess we want to perform the unfreeze regardless of the `key`, perhaps I should just drop this `if`.

Similarly, maybe I should unconditionally wrap all the `convertSet`s, instead of just ones mentioning specific keys? i.e. remove this condition:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/8ae9fc7cc23984ba829a053de8818c07771b0260/src/lib/ikea.ts#L97-L106

Reply:
> I guess we want to perform the unfreeze regardless of the key, perhaps I should just drop this if.
I guess yes, given this is only called for converters which have a key in `keysNeedingUnfreeze`

> Similarly, maybe I should unconditionally wrap all the convertSets, instead of just ones mentioning specific keys? i.e. remove this condition:

I think this is fine for now

Reply:
I am on a trip overseas, but I can look at the PR in 2 weeks or so (if it still makes sense).

Reply:
Thanks - I'll leave it as-is, if this does get merged before then, we can always remove it in a future PR, which will appear to users as adding functionality